src/Microsoft.FeatureManagement.Telemetry.ApplicationInsights.AspNetCore/Constants.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.FeatureManagement.Telemetry.ApplicationInsights.AspNetCore
+{
+    internal class Constants
+    {
+        public const string TargetingIdKey = "TargetingId";
```

Comment:
Is there any reason why we discard the prefix "Microsoft.FeatureManagement" we used before?

Reply:
Before, we were placing it on `HttpContext.Items`. Many different services look at this and store their own things there- which is why it needed the full "Microsoft.FeatureManagement" prefix. 

Now, we're placing it in `RequestTelemetry`. Not only is the scope of this specific to App Insights, but more importantly "TargetingId" was already being placed on it, just later in the code (so this PR isn't introducing a new key). 

It's a reasonable argument to say we should add the prefix to "TargetingId" on all of the outgoing telemetry, which we might do at some point. But that decision should be a separate issue, as it wouldn't just live on "TargetingId" but all of the fields we emit (like "Variant" and "VariantAssignmentReason")

Reply:
I suppose this is the code you refer to
```cs
telemetryWithSupportProperties.Properties[Constants.TargetingIdKey] = requestTelemetry.Properties[Constants.TargetingIdKey];
```

The key on the left doesn't need to be the same key on the right. I agree the key on the left is a bigger discussion, but the key on the right can be just anything. Now that we put the middleware and initializer in the same package, the key on the right can even be something like `static object TargetingIdCacheKey = new object();`.